### PR TITLE
Update COMMENT ON docs for types

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-10455

### CODE REVIEW

**Review by annrpom — APPROVED** [review]
this lgtm!

i have a supernit so feel free to disregard, but what do you think about separating out the 
`Add or remove a comment to or from a type` 
into 
`Add a comment to a type` and `Remove a comment from a type`

so it aligns more with how we do database comment adding/removal? 
With `Remove a comment from a type` being below `Remove a comment from a database` so that the ordering matches

**Comment by rmloveland** [comment]
> this lgtm!

Thanks for the review!
 
> i have a supernit so feel free to disregard, but what do you think about separating out the `Add or remove a comment to or from a type` into `Add a comment to a type` and `Remove a comment from a type`

fixed, not a nit! thanks
 
> so it aligns more with how we do database comment adding/removal? With `Remove a comment from a type` being below `Remove a comment from a database` so that the ordering matches

should be fixed now

thanks again for taking this review!

File: src/current/v24.2/comment-on.md [hunk 1]
```diff
@@ -204,6 +249,50 @@ To remove a comment from a database:
...
+~~~
+
+~~~
+  type | object_id | sub_id | comment
+-------+-----------+--------+-----------
+     7 |       112 |      0 | 3D POINT
+(1 row)
+~~~
+
+To remove a comment from a type, use a statement like the following:
```
Comment (taroface) [inline src/current/v24.2/comment-on.md:289]: I'm not sure everything above this line is needed for this example. It duplicates the "Add a comment to a type" content and kind of buries the relevant line. Usually I say something like `To remove the comment from the type you created in the [preceding example](#add-a-comment-to-a-type)` or just leave it implicit that all the examples on the page are connected.

Comment (taroface) [inline src/current/v24.2/comment-on.md:289]: Not sure if this is the best wording, but maybe best to be explicit:

```suggestion
To remove a comment from a type, add a `NULL` comment:
```

**Review by taroface — APPROVED** [review]
LGTM, but had some suggestions.

File: src/current/v24.2/comment-on.md [hunk 2]
```diff
@@ -204,6 +249,50 @@ To remove a comment from a database:
 (4 rows)
 ~~~
 
+### Remove a comment from a type
+
+Suppose we [create a type]({% link {{ page.version.version }}/create-type.md %}):
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v24.2/comment-on.md:254]: ✅

File: src/current/v24.2/comment-on.md (hunk 1 above)
Comment (rmloveland) [reply src/current/v24.2/comment-on.md:289]: SGTM, based on this and your other suggestion re: this section it's now

> To remove the comment from the type you created in the [preceding example](#add-a-comment-to-a-type), add a `NULL` comment:
> ~~~ sql
> COMMENT ON TYPE my_point IS NULL;
> ~~~

**Pull request merged**